hamocc/mo_powach.F90

```diff
@@ -432,7 +430,7 @@ subroutine powach(kpie,kpje,kpke,kbnd,prho,omask,psao,ptho,lspin)
             K2p = keqb( 9,i,j)
             K3p = keqb(10,i,j)
 
-            call carchm_solve(saln,c,alk,sit,pt,K1,K2,Kb,Kw,Ks1,Kf,Ksi,K1p,K2p,K3p,ah1,ac,niter)
+            call carchm_solve(psao(i,j,kbo(i,j)),c,alk,sit,pt,K1,K2,Kb,Kw,Ks1,Kf,Ksi,K1p,K2p,K3p,ah1,ac)
```

Comment:
why not remaining consistent with what is calculated in the water column? - do we expect the sediment salinity always being between 5 and 40?

Reply:
I don't understand this comment. It was inconsistent before (water column saln=[25,40], sediment saln=[0,40]), now it is consistent. I think a range of [5,40] should be good enough.

Reply:
Hi @JorgSchwinger, ok, I got confused through the doubling above of 
`s = min(saln_max,max(saln_min,psao(i,j,k)))`
which is sometimes also calculated before the call to `carchm_solve` - so my comment above was also due to this confusion.

Reply:
Yes, I agree the doubling is not so nice. I finally put it in, because we also use carchm_solve in the sediment and I didn't see a much better solution.

Reply:
Hi @JorgSchwinger , maybe I am missing something, but we could leave out the 
`s = min(saln_max,max(saln_min,saln))`
in `carchm_kequi` and `carchm_solve` since one can calculate it before (as now also done for the water column - would be needed to put in again in `mo_powach`, which was my initial cause of confusion).